pkgs/applications/audio/magnetophonDSP/VoiceOfFaust/default.nix

```diff
@@ -16,35 +16,20 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ faust2jack faust2lv2 ];
 
-  runtimeInputs = [ pitchTracker ];
+  runtimeInputs = [ pitchTracker jack-example-tools ];
```

Comment:
Is this even necessary, as you seem to hardcode the paths already?

Reply:
Thanks for the feedback.
I removed that line.